README.md

```diff
@@ -26,8 +26,12 @@ Go here to [create an issue](https://github.com/newrelic/docs-website/issues/new
 
 ### Edit a page
 
-If you'd like to get more directly involved, you can make some changes yourself. On any doc, click the **Edit this page** button to get started making a change.
-You can edit a doc directly on the GitHub site. You don't need to actively fork the site or build it locally to help out. This is the easiest way to contribute.
+If you'd like to get more directly involved, you can edit the docs yourself! Here's how:
+
+1. Every doc page on docs.newrelic.com has an **Edit this page** button. Click it to get started making a change. This will take you to GitHub and the source file for that doc page. 
```

Comment:
That edit this page button was not obvious or easy to find. It blends in. I eventually found it. Just some feedback 😄

Reply:
Thanks, that's helpful!